marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/temporal/hub-temporal.sjs

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+const temporal = require("/MarkLogic/temporal.xqy");
+
+module.exports.getTemporalCollections = module.amp(
+  function getTemporalCollections() {
```

Comment:
I think it'd be better for this to just call temporal.collections(). Returning an object with each collection as the key and a value of "true" is very specific to what invoke-queue-write wants. invoke-queue-write should be responsible for doing that; this should just call temporal.collections() and return the result.